en/identity-server/next/docs/includes/db-config-table.md

```diff
@@ -38,3 +38,29 @@ The elements in the above configuration are described below:
 !!! info 
     For more information on other parameters that can be defined in
     the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat JDBC Connection Pool](http://tomcat.apache.org/tomcat-9.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+
+??? note "Support for case-sensitive usernames"
+
+    WSO2 Identity Server supports case-insensitive usernames by default. Hence, if you need to have case-sensitive usernames, you need to configure the following properties.
+    
+    For the primary user store, add the following configurations to the `<IS-HOME>/repository/conf/deployment.toml` file.
```

Comment:
structure this similar to the notes above.

Reply:
https://github.com/wso2/docs-is/pull/4209/commits/d0b97b1a597730a4d099c4dc1565e31452c0f03f